.circleci/config.yml

```diff
@@ -68,3 +112,13 @@ workflows:
       - test_feature_engine_py37
       - test_feature_engine_py38
       - test_style
+      - package_and_upload_to_pypi:
```

Comment:
@ChristopherGS will all tests run on every branch? but only what is in master published and uploaded? see below.